packages/block-library/src/cover/edit/index.js

```diff
@@ -318,7 +318,19 @@ function CoverEdit( {
 	const blockProps = useBlockProps( { ref } );
 
 	// Check for fontSize support before we pass a fontSize attribute to the innerBlocks.
```

Comment:
Should we update this comment to reflect recent changes? There's a bit more going on now than just checking fontSize support.

Reply:
Oh, yes, of course, I'll update this tomorrow 👍